### PR TITLE
Allowing additional headers from Context

### PR DESCRIPTION
**Pull request by Gabriel-AL**
Hello !

I've been in need for additional headers to be sent with the requests to my shopware app, so here is my suggestion !

Thank you for your work, and have a nice day

### CODE REVIEW

**Comment by vienthuong** [comment]
Thanks for your contribution! Have a niceday!

File: src/Service/ApiService.php [hunk 1]
```diff
@@ -62,11 +62,11 @@ protected function getBasicHeaders(array $additionalHeaders = []): array
             throw new \Exception('Please call setContext method before sending the request');
         }
 
-        $basicHeaders = [
+        $basicHeaders = array_merge([
             'Accept' => $this->contentType,
             'Authorization' => 'Bearer ' . $context->accessToken->accessToken,
             'Content-Type' => 'application/json'
-        ];
+        ], $this->context->additionalHeaders);
```
Comment (raphael-homann) [inline src/Service/ApiService.php:69]: $context should be used, phpstan would be happier

**Review by raphael-homann — COMMENTED** [review]
just a phpstan fix, with no impact

**Review by raphael-homann — COMMENTED** [review]
just a phpstan fix, with no impact

**Pull request merged**